docs/150-git.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Git
```

Comment:
can we put this section under the environment section?

there are already some github-related docs there, maybe it makes sense for them to be together?

Reply:
Hm. To me, the environment area looks like the bare setup to get started and further customization comes below.

On the other hand it's a really small setting and is no really worth its own section.

I will move it over then.

Reply:
thanks :)